bindings/nodejs/lib/types/wallet/index.ts

```diff
@@ -8,7 +11,7 @@ export * from './output';
 export * from './outputParams';
 export * from './participation';
 export * from './preparedMintTokenTransactionData';
-export * from './preparedTransactionData';
+export * from './preparedTransaction';
```

Comment:
Should we also remove the Data from `preparedMintTokenTransactionData` then?

Reply:
yes